lib/active_merchant/billing/gateways/paypal/paypal_checkout_common.rb

```diff
@@ -0,0 +1,115 @@
+module ActiveMerchant
+  module Billing
+    module PaypalCheckoutCommon
+
+      URLS = {
+        :test_url     => "https://api.sandbox.paypal.com",
+        :live_url     => "https://api.paypal.com"
+      }
+
+      ALLOWED_INTENT              = %w(CAPTURE AUTHORIZE).freeze
+      ALLOWED_PAYMENT_TYPE        = %w(ONE_TIME RECURRING UNSCHEDULED).freeze

```

Comment:
what does UNSCHEDULED do?

Reply:
Removed them as the recurring API supported by PayPal is not used in our integration.

Reply:
is it ONE_TIME by default?